packages/wallet/test/hardware/ledger/LedgerKeyAgent.test.ts

```diff
@@ -298,6 +314,317 @@ describe('LedgerKeyAgent', () => {
+
+            expect(await tx.sign()).toBeTruthy();
+          });
+
+          it('would throw while trying to sign a transaction with UnregisterDelegateRepresentative certs', async () => {
+            const txBuilder = wallet.createTxBuilder();
+            txBuilder.partialTxBody.certificates = [
+              {
+                __typename: Cardano.CertificateType.UnregisterDelegateRepresentative,
+                dRepCredential: {
+                  hash: Crypto.Hash28ByteBase16.fromEd25519KeyHashHex(dRepKeyHash),
+                  type: Cardano.CredentialType.ScriptHash
+                },
+                deposit: 5n
+              }
+            ];
+            const tx = txBuilder
+              .addOutput(
+                txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+              )
+              .build();
+
+            await expect(tx.sign()).rejects.toThrow(
+              InvalidDataReason.SIGN_MODE_ORDINARY__CERTIFICATE_DREP_CREDENTIAL_ONLY_AS_PATH
+            );
+          });
+
+          it('can sign a transaction with UpdateDelegateRepresentative certs', async () => {
+            const txBuilder = wallet.createTxBuilder();
+            txBuilder.partialTxBody.certificates = [
+              {
+                __typename: Cardano.CertificateType.UpdateDelegateRepresentative,
+                anchor: null,
+                dRepCredential: {
+                  hash: Crypto.Hash28ByteBase16.fromEd25519KeyHashHex(dRepKeyHash),
+                  type: Cardano.CredentialType.KeyHash
+                }
+              }
+            ];
+            const tx = txBuilder
+              .addOutput(
+                txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+              )
+              .build();
+
+            expect(await tx.sign()).toBeTruthy();
+          });
+
+          it('would throw while trying to sign a transaction with UpdateDelegateRepresentative certs', async () => {
+            const txBuilder = wallet.createTxBuilder();
+            txBuilder.partialTxBody.certificates = [
+              {
+                __typename: Cardano.CertificateType.UpdateDelegateRepresentative,
+                anchor: null,
+                dRepCredential: {
+                  hash: Crypto.Hash28ByteBase16.fromEd25519KeyHashHex(dRepKeyHash),
+                  type: Cardano.CredentialType.ScriptHash
+                }
+              }
+            ];
+            const tx = txBuilder
+              .addOutput(
+                txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+              )
+              .build();
+
+            await expect(tx.sign()).rejects.toThrow(
+              InvalidDataReason.SIGN_MODE_ORDINARY__CERTIFICATE_DREP_CREDENTIAL_ONLY_AS_PATH
+            );
+          });
+        });
+        // describe('multisig tx mode', () => {});
```

Comment:
This is not supported for now and you left just a placeholder to not forget or it is a TODO and should be in the scope of this PR?

Reply:
removed here https://github.com/input-output-hk/cardano-js-sdk/pull/1211/commits/0f6bc6de2d8009ed43f47f4cf3a7b1f222160349